conda-recipe/meta.yaml

```
@@ -20,7 +20,7 @@ requirements:
     - future
     - h5py
     - pyopengl
-    - pyqt >=5.6
+    - pyqt >=5.6,<=5.12.*
```

```suggestion
    - pyqt >=5.6,<=5.12
```
this might be a conda/mamba bug (there will be a warning that the `.*` is superfluous and deprecated but I would still imagine it would work - it doesn't, that's why the tests didn't run through on that one